src/hooks/useConnectionForm.ts

```diff
@@ -96,8 +96,12 @@ export const useConnectionForm = (validationSchema: ZodObject<ZodRawShape>, mode
 		integrationName?: string
 	) => {
 		const connectionData = flattenFormData(getValues(), formSchema);
+		let formattedIntegrationName = integrationName;
+		if (integrationName && isGoogleIntegration(stripGoogleConnectionName(integrationName) as Integrations)) {
+			formattedIntegrationName = defaultGoogleConnectionName;
+		}
```

Comment:
without `let`, like example:
```suggestion
		const formattedIntegrationName = integrationName && isGoogleIntegration(stripGoogleConnectionName(integrationName) as Integrations)
  ? defaultGoogleConnectionName
  : integrationName;
```

Reply:
done